prov/efa/src/rxr/rxr_read.h

```diff
@@ -79,6 +79,9 @@ struct rxr_read_entry *rxr_read_alloc_entry(struct rxr_ep *ep, int entry_type, v
 void rxr_read_release_entry(struct rxr_ep *ep, struct rxr_read_entry *read_entry);
 
 /* used by read message protocol and read write protocol */
+int rxr_locate_iov_pos(struct iovec *iov, int iov_count, size_t offset,
```

Comment:
This should probably move to a common file somewhere.

Reply:
Yes, I am thinking to move it to ofi_iov.h, but that would be a separate PR.